india_compliance/gst_india/setup/property_setters.py

```diff
@@ -111,6 +105,21 @@ def get_property_setters():
     ]
 
 
+def get_default_property_setters():
```

Comment:
why is this a function. I don't see any kind of computation happening here. recommending a constant like `TRANSPORTER_NAME_PROPERTIES` instead. suggested constant name: `DEFAULT_PROPERTIES`.

it should then be optionally sent with the return value of `get_property_setters` based on a flag called something like `include_defaults` (flag should default to False).